tensilelite/Tensile/Source/lib/include/Tensile/DataTypes_Float8_BFloat8.hpp

```diff
@@ -629,13 +642,16 @@ namespace Tensile
         // check for nan
         inline HIP_HOST_DEVICE bool is_nan() const
         {
+#if defined(HIP_FP8_TYPE_OCP)
             if(IsOCPSupported())
             {
                 return (T == hip_f8_type::fp8)   ? ((data & 0x7f) == 0x7f)
                        : (T == hip_f8_type::bf8) ? ((data & 0x7f) > 0x7c)
                                                  : false;
             }
-            else if(get_hip_f8_bias_mode())
+            else
+#endif
+                if(get_hip_f8_bias_mode())
```

Comment:
Not important but you may use { } to make the blank alinged. For example, 
            else
#endif
            {
                if(get_hip_f8_bias_mode())
                {
                    return data == 0x80;
                }
            }

Reply:
done